src/libraries/Microsoft.PowerFx.Connectors/Execution/FormulaValueSerializer.cs

```diff
@@ -176,6 +183,10 @@ private void WriteProperty(string propertyName, OpenApiSchema propertySchema, Fo
                     {
                         WriteDecimalValue(decimalValue.Value);
                     }
+                    else if (fv is BlankValue)
+                    {
+                        WriteDecimalValue(0);

```

Comment:
This can be dangerous. Empty is not the same as 0, Null is not the same as 0. In c# you can have a nullable property int? and having Null and 0 gives two different meaning. The same applies to the boolean false value or empty arrays.

Imho, we should probably just skip writing properties all together

Reply:
agreed - we could do a top-level check for Blank. 

Also - what about ErrorValue?  We probably need to treat that like blank too (I think that's what we do in SQL case, which also can't represent error)